plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -804,5 +804,33 @@ SecRule REQUEST_FILENAME "@contains /config/apps/ransomware_protection/extension
     ctl:ruleRemoveTargetById=932130;ARGS:value,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0'"
 
+#
+# [ Nextcloud Notes ]
+#
+
+# Note name and content can be anything
+SecRule REQUEST_FILENAME "@rx /index.php/apps/notes/api/v[0-9]/notes" \
```

Comment:
The dot (`.`) has to be escaped, that was correct, i meant only escaping of `/`.

Reply:
I've escaped ``.`` now